nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/proxy/ConanProxyHelper.groovy

```diff
@@ -40,15 +40,19 @@ import static org.sonatype.repository.conan.internal.metadata.ConanMetadata.VERS
  */
 class ConanProxyHelper
 {
-  public static final List<AssetKind> DOWNLOAD_ASSET_KINDS = ImmutableList.of(AssetKind.DOWNLOAD_URL, AssetKind.DIGEST, AssetKind.CONAN_PACKAGE_SNAPSHOT)
+  public static final List<AssetKind> METADATA_ASSET_KINDS = ImmutableList.of(
+      AssetKind.DOWNLOAD_URL,
+      AssetKind.DIGEST,
+      AssetKind.CONAN_PACKAGE_SNAPSHOT
+  )
```

Comment:
I do not agree with this change. Conanmanifest or Conaninfo is also metadata therefore we are adding more confuse here. Mayby URL_ASSET_KIND(not sure is it true for CONAN_PACKAGE_SNAPSHOT) or DOWNLOAD_METADATA_ASSET_KINDS. Or leave it as it.

Reply:
I returned back prev name
https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/98/commits/3c7d54608b90e0730d43d62d549032b70bb4d380